vm/Cargo.toml

```diff
@@ -114,35 +119,33 @@ winreg = "0.55"
 [target.'cfg(windows)'.dependencies.windows]
 version = "0.52.0"
 features = [
-  "Win32_Foundation",
-  "Win32_System_LibraryLoader",
-  "Win32_System_Threading",
-  "Win32_System_Time",
-  "Win32_UI_Shell",
+    "Win32_Foundation",
+    "Win32_System_LibraryLoader",
+    "Win32_System_Threading",
+    "Win32_UI_Shell",
 ]
 
 [target.'cfg(windows)'.dependencies.windows-sys]
 workspace = true
 features = [
-  "Win32_Foundation",
-  "Win32_Globalization",
```

Comment:
Globalization is needed for `_winapi`

Reply:
Also `Win32_System_Time` is needed from the `windows` crate for other things I think.

Reply:
Yeah, it was a bad merge.